f2/src/utils/submitReportToServer.js

```diff
@@ -1,16 +1,25 @@
+var fs = require('fs')
 const flatten = require('flat')
 const fetch = require('isomorphic-fetch')
 const FormData = require('form-data')
 const { formDefaults } = require('../forms/defaultValues')
 
 async function submitReportToServer(url = '', data = {}) {
   const flattenedData = flatten(data, { safe: true })
+  console.log(flattenedData)
```

Comment:
useful for getting the data fields for the k6 scripts. k6 didn't want to include `formDefaults` in the script unfortunately. Nor `flatten`, for that matter. 😞